### PR TITLE
fixes #874

### LINKED ISSUE #874: citation erata 2
Very very randomly,

I was looking at another list and it's source iis cited as https://concepticon.clld.org/sources/matisoff1978. This item is available online at: https://stedt.berkeley.edu/pubs_and_prods/Matisoff_1978_Variational-Semantics-TB.pdf

which could be added to your citation, along with the book's series (Occasional papers of the Wolfenden Society on Tibeto-Burman Linguistics vol.VI), and the book's location of publisher (Philadelphia, USA), and the book even has an ISBN: 0-915980-85-1

### PR DESCRIPTION
**Pull request by SimonGreenhill**
Fills in the details for Matisoff 1978, #874

### CODE REVIEW

File: concepticondata/references/references.bib [hunk 1]
```diff
@@ -1705,7 +1705,13 @@ @Book{Matisoff1978
   Title                               = {{V}ariational semantics in {T}ibeto-{B}urman. {T}he 'organic' approach to linguistic comparison},
   Author                              = {Matisoff, James A.},
   Year                                = {1978},
-  Publisher                           = {Institute for the Study of Human Issues}
+  Publisher                           = {Institute for the Study of Human Issues},
+  Pages                               = {xviii+331},
+  Address                             = {Occasional Papers of the Wolfenden Society on Tibeto-Burman Linguistics, Volume VI. Philadelphia},
+  Publisher                           = {Philadelphia: Institute for the Study of Human Issues},
```
Comment (LinguList) [inline concepticondata/references/references.bib:1711]: Two publishers here, I think this should be reduced to one, right?

File: concepticondata/references/references.bib [hunk 2]
```diff
@@ -1705,7 +1705,13 @@ @Book{Matisoff1978
   Title                               = {{V}ariational semantics in {T}ibeto-{B}urman. {T}he 'organic' approach to linguistic comparison},
   Author                              = {Matisoff, James A.},
   Year                                = {1978},
-  Publisher                           = {Institute for the Study of Human Issues}
+  Publisher                           = {Institute for the Study of Human Issues},
+  Pages                               = {xviii+331},
+  Address                             = {Occasional Papers of the Wolfenden Society on Tibeto-Burman Linguistics, Volume VI. Philadelphia},
```
Comment (LinguList) [inline concepticondata/references/references.bib:1710]: Address is not the right one, as this is the series.

Comment (SimonGreenhill) [reply concepticondata/references/references.bib:1710]: yeah, that's what glottolog had, so I presumed it was some quirk of bibtex that I didn't know. I'll fix tomorrow and PR.

File: concepticondata/references/references.bib (hunk 1 above)
Comment (SimonGreenhill) [reply concepticondata/references/references.bib]: done

File: concepticondata/references/references.bib (hunk 2 above)
Comment (SimonGreenhill) [reply concepticondata/references/references.bib]: done

**Review by LinguList — APPROVED** [review]

**Pull request merged**